website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -78,12 +78,23 @@ Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt
 
    - To find your project ID, select **Develop** in the dbt Cloud navigation menu. You can use the URL to find the project ID. For example, in `https://cloud.getdbt.com/develop/26228/projects/123456`, the project ID is `123456`.
 
+### Set environment variables
+
+To set environment variables in the dbt Cloud CLI for your dbt project:
+
+1. Select the gear icon on the upper right of the page.
+2. Then select **Profile Settings**, then **Credentials**.
+3. Click on your project and scroll to the **Environment Variables** section.
+4. Click **Edit** on the lower right and then set the user-level environment variables.  
+
+If you're receiving an `Env var required but not provided: 'DBT_ENVIRONMENT'` error, you may be unintentionally running dbt Core. You can verify this by running `which dbt` to confirm if the output refers to the dbt Cloud CLI.
```

Comment:
I don't think sentence is correct - this could either be that they're running core or have not added the env variable. I'd strike.